test/unit/sitemap/azure/helix-query.yaml

```diff
@@ -4,7 +4,7 @@ indices:
   blog-posts:
     source: html
     fetch: https://{repo}-{owner}.project-helix.page/{path}
-    target: azure
+    target: azure:default
```

Comment:
@trieloff @dominique-pfister I needed to change this to match the new schema....